spec/serializers/current_user_serializer_spec.rb

```diff
@@ -150,6 +150,27 @@
     end
   end
 
+  describe "#can_ignore_users" do
+    let(:guardian) { Guardian.new(user) }
+    let(:payload) { serializer.as_json }
+
+    context "when user is a regular one" do
+      let(:user) { Fabricate(:user) }
+
+      it "return false for regular users" do
+        expect(payload[:can_ignore_users]).to eq(false)
+      end
+    end
+
+    context "when user is a staff member" do
+      let(:user) { Fabricate(:admin) }
```

Comment:
```suggestion
      let(:user) { Fabricate(:moderator) }
```

Small one but it will be better to test with moderator since it is still a staff user but has less permission than an admin user.